functionary/functionary/settings/django_.py

```diff
@@ -26,9 +26,12 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = bool(util.strtobool(os.environ.get("DEBUG", "false")))
 
-
-ALLOWED_HOSTS = []
-
+# Required if DEBUG is False
+ALLOWED_HOSTS_ENV = os.environ.get("ALLOWED_HOSTS")
+if ALLOWED_HOSTS_ENV is None:
+    ALLOWED_HOSTS = []
+else:
+    ALLOWED_HOSTS = list(map(lambda host: host.strip(), ALLOWED_HOSTS_ENV.split(",")))
```

Comment:
How to meaningfully expose this setting to an end user remains TBD, but I realized it wasn't possible to run with DEBUG=False, so I added a way to at least set this.